lib/index.js

```diff
@@ -185,7 +179,7 @@ export function configureUnifiedLanguageServer(
           } else {
             resolve((context && context.files) || [])
           }
-          /* c8 ignore end */
+          /* c8 ignore stop */
```

Comment:
this was hiding missing coverage for all lines under it

Reply:
Oops, my bad! That was a pretty big mistake then

Reply:
yeah, would be nice if c8 was clearer about this stuff 😅

Reply:
Related: https://github.com/remarkjs/remark-lint/issues/277